extensions/wikia/UserTools/templates/UserTools_Menu.php

```diff
@@ -10,7 +10,7 @@
 	<span class="arrow-icon-ctr"><span class="arrow-icon arrow-icon-single"></span></span>
 	<a href="#"><?= htmlspecialchars( $item['caption'] ); ?></a>
 	<ul id="my-tools-menu" class="tools-menu">
-		<?= F::app()->renderView( 'UserTools', 'Menu', array( 'format' => 'html', 'items' => $item['items'] ) ); ?>
+		<?= F::app()->renderView( 'UserTools', 'Menu', [ 'format' => 'html', 'type' => 'submenu', 'items' => $item['items'] ] ); ?>
```

Comment:
A little bit of docs saying what are the possible types are (submenu/main) would be nice. :)